app/assets/javascripts/catalog_manager/form.js.coffee

```diff
@@ -171,13 +171,11 @@ $ ->
     event.preventDefault()
     if confirm (I18n['catalog_manager']['organization_form']['user_rights']['remove_confirm'])
       identity_id = $(this).data('identity-id')
+      organization_id = $(this).data('organization-id')
       $.ajax
         type: 'POST'
-        url: 'catalog_manager/organizations/remove_fulfillment_rights_row'
-        data:
-          fulfillment_rights:
-            identity_id: identity_id
-            organization_id: $(this).data('organization-id')
+        url: "/catalog_manager/organizations/remove_fulfillment_rights_row?fulfillment_rights[identity_id]=#{identity_id}&fulfillment_rights[organization_id]=#{organization_id}"
```

Comment:
Why move the parameters to the URL? The AJAX data hash is intended to be used for additional params.

Reply:
Specs failed when the params were passed in the hash. So I added them to the URL.

Reply:
Huh that's strange. That works then.